xline/src/client/errors.rs

```diff
@@ -13,6 +13,9 @@ pub enum ClientError {
     /// Propose error
     #[error("propose error {0}")]
     ProposeError(#[from] curp::error::ProposeError),
+    /// Bincode error
+    #[error("bincode error {0}")]
+    BincodeError(#[from] bincode::Error),
```

Comment:
May duplicate with ParseError, please double check it.

Reply:
I think it is not duplicate, BincodeError is an error when encode the command, but ParseError is an error when the response is not the expected response type

Reply:
We should not expose implementation detail such as Bincode to the client.